screens/Profile/ProfileScreen.tsx

```diff
@@ -58,20 +58,32 @@ export const ProfileScreen: React.FC<MainRouteProps> = (props) => {
           title={controller.alertMsg}
         />
         <EditableListItem
-          label={t('name')}
-          value={controller.name}
+          title={t('name')}
```

Comment:
Profile Screen is unused, it is stale code in inji, no need any changes here, as we should be removing this file

Reply:
I haven't removed Profile related code with assumption that we might leverage this structure with upcoming features.
Made changes so that it aligns with current implementation